__tests__/distributors/zulu.test.ts

```diff
@@ -0,0 +1,525 @@
+      },
+      { version: '8.0', arch: 'x86', packageType: 'jdk' }
+    ],
+    [
+      '11.0',
+      {
+        version: '11.0.10',
+        url: 'https://fake.cdn.azul.com/zulu/bin/zulu11.45.27-ca-jdk11.0.10-macosx_x64.tar.gz'
+      },
+      { version: '11.0', arch: 'x86', packageType: 'jdk' }
+    ],
+    [
+      '8',
+      {
+        version: '8.0.282',
+        url: 'https://fake.cdn.azul.com/zulu/bin/zulu8.52.0.23-ca-fx-jdk8.0.282-macosx_x64.tar.gz'
+      },
+      { version: '8', arch: 'x86', packageType: 'jdk+fx' }
+    ],
+    [
+      '11',
+      {
+        version: '11.0.10',
+        url: 'https://fake.cdn.azul.com/zulu/bin/zulu11.45.27-ca-fx-jdk11.0.10-macosx_x64.tar.gz'
+      },
+      { version: '11', arch: 'x86', packageType: 'jdk+fx' }
+    ],
+    [
+      '15',
+      {
+        version: '15.0.2',
+        url: 'https://fake.cdn.azul.com/zulu/bin/zulu15.29.15-ca-jdk15.0.2-macosx_x64.tar.gz'
+      },
+      { version: '15', arch: 'x86', packageType: 'jdk' }
+    ],
+    [
+      '15-ea',
+      {
+        version: '15.0.0',
+        url: 'https://fake.cdn.azul.com/zulu/bin/zulu15.0.25-ea-jdk15.0.0-ea.11-macosx_x64.tar.gz'
+      },
+      { version: '15-ea', arch: 'x86', packageType: 'jdk' }
+    ],
+
+    // jre
+
+    [
+      '8',
+      {
+        version: '8.0.282',
+        url: 'https://fake.cdn.azul.com/zulu/bin/zulu8.52.0.23-ca-jre8.0.282-macosx_x64.tar.gz'
+      },
+      { version: '8', arch: 'x86', packageType: 'jre' }
+    ],
+    [
+      '11',
+      {
+        version: '11.0.10',
+        url: 'https://fake.cdn.azul.com/zulu/bin/zulu11.45.27-ca-jre11.0.10-macosx_x64.tar.gz'
+      },
+      { version: '11', arch: 'x86', packageType: 'jre' }
+    ],
+    [
+      '8.0',
+      {
+        version: '8.0.282',
+        url: 'https://fake.cdn.azul.com/zulu/bin/zulu8.52.0.23-ca-jre8.0.282-macosx_x64.tar.gz'
+      },
+      { version: '8.0', arch: 'x86', packageType: 'jre' }
+    ],
+    [
+      '11.0',
+      {
+        version: '11.0.10',
+        url: 'https://fake.cdn.azul.com/zulu/bin/zulu11.45.27-ca-jre11.0.10-macosx_x64.tar.gz'
+      },
+      { version: '11.0', arch: 'x86', packageType: 'jre' }
+    ],
+    [
+      '8',
+      {
+        version: '8.0.282',
+        url: 'https://fake.cdn.azul.com/zulu/bin/zulu8.52.0.23-ca-fx-jre8.0.282-macosx_x64.tar.gz'
+      },
+      { version: '8', arch: 'x86', packageType: 'jre+fx' }
+    ],
+    [
+      '11',
+      {
+        version: '11.0.10',
+        url: 'https://fake.cdn.azul.com/zulu/bin/zulu11.45.27-ca-fx-jre11.0.10-macosx_x64.tar.gz'
+      },
+      { version: '11', arch: 'x86', packageType: 'jre+fx' }
+    ],
+    [
+      '15',
+      {
+        version: '15.0.2',
+        url: 'https://fake.cdn.azul.com/zulu/bin/zulu15.29.15-ca-jre15.0.2-macosx_x64.tar.gz'
+      },
+      { version: '15', arch: 'x86', packageType: 'jre' }
+    ]
+  ])('version is %s -> %o', async (functionInput, expected, constructorInput) => {
```

Comment:
1) where `functionInput` is used?
2) Why do we need to compare urls? Is it not enough to compare version?

Reply:
Done. Thank you.